wheel_radius: 13.0